src/platform/forms-system/src/js/web-component-patterns/titlePattern.js

```diff
@@ -23,11 +23,17 @@ import React from 'react';
 export const titleUI = (title, description) => {
   return {
     'ui:title': (
-      <h3 className="vads-u-color--gray-dark vads-u-margin-y--0">{title}</h3>
+      <>
+        <h3 className="vads-u-color--gray-dark vads-u-margin-top--0 vads-u-margin-bottom--3">
+          {title}
+        </h3>
+        {description && (
+          <span className="vads-u-margin-bottom--0 vads-u-font-family--sans vads-u-font-weight--normal vads-u-font-size--base vads-u-line-height--4 vads-u-display--block">
+            {description}
+          </span>
+        )}
+      </>
```

Comment:
@jeana-adhoc This all looks great. Just to make sure I understand, you want all forms we've made using titleUI to now have the heading and the descriptive text be inside the legend? There's none that need to be in a different state than this?

Reply:
Correct. I think UI:description was (or should now be) intended to be text in a legend. Any additional text in a form that's not meant for the legend can just be normal `<p>` text